js/packages/phoenix-client/src/__generated__/api/v1.ts

```diff
@@ -3527,6 +3527,7 @@ export interface operations {
                     inputs: Record<string, unknown>[];
                     outputs?: Record<string, unknown>[];
                     metadata?: Record<string, unknown>[];
+                    splits?: Record<string, unknown>[];
```

Comment:
What does the payload look like here. what is the key and what is the value of each record? feels like it doesn't need to be unknown given splits are strings??